battle-service/src/test/java/by/ibondarau/tictactoe/battleservice/checker/impl/GameCheckerImplTest.java

```diff
@@ -0,0 +1,89 @@
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(1).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(2).setSecondCoordinate(2));
+
+        Assertions.assertTrue(gameChecker.checkWin(firstPlayerId, 3, moves));
+    }
+
+
+    @Test
+    void testAddMoveWinRow() {
+        List<Move> moves = new LinkedList<>();
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(0).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(1).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(0).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(1).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(2).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(1).setSecondCoordinate(2));
+
+        Assertions.assertTrue(gameChecker.checkWin(secondPlayerId, 3, moves));
+    }
+
+
+    @Test
+    void testAddMoveWinColumn() {
+        List<Move> moves = new LinkedList<>();
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(0).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(1).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(1).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(0).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(2).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(2).setSecondCoordinate(1));
+
+        Assertions.assertTrue(gameChecker.checkWin(secondPlayerId, 3, moves));
+
+    }
+
+    @Test
+    void testAddMoveWinAntiDiagonal() {
+        List<Move> moves = new LinkedList<>();
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(0).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(2).setSecondCoordinate(0));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(1).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(1).setSecondCoordinate(1));
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(2).setSecondCoordinate(2));
+        moves.add(new Move().setPlayerId(secondPlayerId).setFirstCoordinate(0).setSecondCoordinate(2));
+
+        Assertions.assertTrue(gameChecker.checkWin(secondPlayerId, 3, moves));
+    }
+
+    @Test
+    void testDraw() {
+
+        List<Move> moves = new LinkedList<>();
+        moves.add(new Move().setPlayerId(firstPlayerId).setFirstCoordinate(0).setSecondCoordinate(0));
```

Comment:
It make sense to create helper class i.e. MoveHelper which could be utilize for many purposes. 
MoveHelper.createMoveAtCoordinate(String playerId,int firstCoordinate,int secondCoordinate)
or potentially could be expanded to createwhole list of moves.

Reply:
I don't think this is really necessary thing to do since the helper line isn't much shorter then the original one)  Besides, I don't create moves manually inside this application, so that would be test-only tool. Finally, that would require rewriting many tests. Although I'm not totally against the idea of creating a helper class for testing purposes, I don't think this is necessary at this point.

Reply:
I refactored some tests and created a cool helper class, I hope you like it